lighthouse-cli/test/fixtures/screenshots.html

```diff
@@ -0,0 +1,17 @@
+<style>
+body {
+  background: lightblue url('https://i.stack.imgur.com/YbUvO.gif') no-repeat fixed center;
```

Comment:
imgur definitely goes down at times. Can we just generate a high entropy image really quickly and check it in?

Reply:
seems rare https://status.imgur.com/history